subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportViewerQuery.kt

```diff
@@ -13,7 +13,9 @@ class ReportViewerQuery {
     private val encoder by lazy { Base64.getEncoder() }
 
     /**
-     * пример ответа: {"filter":{"error":1,"fail":1,"groups":["messenger"],"other":1}}
+     * Result example:
+     *
+     * {"filter":{"error":1,"fail":1,"groups":["messenger"],"other":1}}
```

Comment:
Let's use code blocks:

```kotlin
/**
*
* ```gson
* {"filter":{"error":1,"fail":1,"groups":["messenger"],"other":1}}
* ```
*/
```

Reply:
I applied but it gave nothing

Reply:
I see the difference in IDE.
Maybe it will affect more in a generated HTML.